merlin-server/sql/merlin/tables/activity_presets.sql

```diff
@@ -0,0 +1,71 @@
+create table activity_presets(
+  id integer generated always as identity primary key,
+  model_id integer not null,
+  name text,
```

Comment:
Just pattern matching - any reason for `name` to be nullable?

Reply:
Not that I can think of. I've added the `not null` constraint.